native/src/common.cpp

```diff
@@ -1,6 +1,7 @@
 #include "CommonCollections.h"
 #include "Logging.h"
 #include "commonOsmAndCore.h"
+#include "json.hpp"
```

Comment:
You include new hpp file,
Will be exception during linking for iOS (core/CMakeLists.txt)

Reply:
Max didn't reproduce possible problems

Reply:
Hpp is a header file looks like so it's not necessary to add